documentation/1.9/content/concepts/model.md

```diff
@@ -262,6 +262,18 @@ This feature does not apply to named security providers within a realm. These it
 
 This feature cannot be use to undeploy applications or remove libraries.
```

Comment:
I checked, this is no longer true, apps and libraries can be removed this way.

Reply:
changed to "can"